indico/htdocs/sass/modules/_event_display.scss

```diff
@@ -100,32 +100,81 @@ li > table + .note-area-wrapper {
         font-size: 1.1em;
     }
 }
+.participant-list-wrapper{
+    position: relative;
+
+    &.collapsible{
+        &.collapsed {
+            max-height: 4.75em;
+            overflow: hidden;
+
+            .gradient-layer {
+                @include gradient(rgba(0,0,0,0.0), #ECECEC);
+                display: block;
+                position: absolute;
+                bottom: 0;
+                height: 1.5em;
+                width: 100%;
+                cursor: pointer;
+            }
+        }
 
-ul.participant-list {
-    @extend %font-family-title;
+        &:hover {
+            & > .trigger {
+                color: $light-black;
+            }
+        }
 
-    padding: 0;
-    margin: 0;
-    list-style-type: none;
+        & > .trigger {
+            display: inline-block;
+            position: absolute;
+            cursor: pointer;
+            right: 0.6em;
+            top: 0;
+            color: $dark-gray;
+            @include user-select(none);
+        }
 
-    li {
-        @include border-radius(2px);
-        @include icon-before('icon-user');
+        & > ul.participant-list {
+            width: 95%;
+        }
+    }
 
-        display: inline-block;
-        font-size: 0.95em;
-        padding: 0.2em 0.5em;
-        margin-bottom: 0.1em;
-        color: $black;
-        background: $pastel-gray;
+    .gradient-layer, .trigger {
+        display: none;
+    }
 
-        &:before {
-            color: $gray;
-            margin-right: 0.4em;
-        }
+    .height-control {
+        height: 5.3em;
```

Comment:
I think this deserves a comment. Nobody will know what this is used for in a month or so.